README.md

```diff
@@ -19,6 +19,34 @@ def deps do
 end
 ```
 
+## Usage
+```elixir
+defmodule Membrane.Demo.FlacPipeline do
+  use Membrane.Pipeline
+
+  @impl true
+  def handle_init(_opts) do
+    children = %{
+      file: %Membrane.File.Source{location: "sample.flac"},
+      parser: %Membrane.FLAC.Parser{streaming?: false},
+      sink: %Membrane.File.Sink{location: "out.flac"}
```

Comment:
I suppose that input and output files will be exactly the same so instead of `File.Sink` let's use [`Fake.Sink`](https://github.com/membraneframework/membrane_fake_plugin). This will suggest that there should be other elements after parser that do something with parsed data.